docs/notebooks/MetaOT.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<p>Am a bit too picky, but for consistency, let's use <code>\varepsilon</code>  .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>